.circleci/config.yml

```diff
@@ -55,20 +55,90 @@ jobs:
             pip install --upgrade pip setuptools
             pip install -r requirements.txt
             pip install -r deid/requirements.txt
+            pip install -r dev_requirements.txt
       - run:
           name: Show bashrc file
           command: cat ${BASH_ENV}
       # Test
       #   This would typically be a build job when using workflows, possibly combined with build
       - run:
           name: Run unit tests
+          working_directory: ~/curation
+          command: |
```

Comment:
What is the development flow where someone would only trigger this, and not the integration tests?

In my mind, the usual flow would be:
- push feature branch (starts running CI)
- create PR (attaches to existing CI run)

So based on my understanding, this would just prevent tests from running in the scenario where someone pushed up a branch but either didn't create a PR, or made additional pushes before creating a PR.

Is there a reason this person wants unit tests to run?

Reply:
Yes, there is a reason to run unit tests on each commit.  Unit tests should be fast, therefore, inexpensive in time.  If a developer is updating a portion of code, s/he can make breaking changes (particularly in error prone code).  Unit tests (done properly) will help identify breaking changes before getting to the PR phase.  Because some features require more than one commit to complete, having that information sooner rather than later is useful.

Reply:
That hasn't been the workflow because prior to this, all integration and unit tests were written in the same test package (some in the same module) module and required access to BigQuery services.  While I highly advocate for running unit tests before checking in code, I know it isn't done in practice.  This PR should make is possible to run unit tests locally.  But I don't expect current habits to change.